doc/users/next_whats_new/steppatch_and_stairs.rst

```diff
@@ -0,0 +1,22 @@
+New `~.matplotlib.patches.StepPatch` artist and a `.pyplot.stairs` method
```

Comment:
Do we normally link to the pyplot version or the axes version?

Reply:
You tell me

Reply:
At least step_demo is using it